compiler/damlc/tests/src/DA/Test/ScriptService.hs

```diff
@@ -659,7 +659,41 @@ main =
                 expectScriptSuccess rs (vr "testSucceed") $ \r ->
                   matchRegex r "Active contracts:  #0:0, #1:0"
                 expectScriptFailure rs (vr "testFail") $ \r ->
-                  matchRegex r "missing authorization from 'p1'"
+                  matchRegex r "missing authorization from 'p1'",
+              testCase "submitTree" $ do
```

Comment:
I finally got annoyed enough to open an issue to improve our testing infrastructure here https://github.com/digital-asset/daml/issues/8753